bootstrap.sh

```diff
@@ -2,11 +2,19 @@
 
 # Installing packages
 apt update
-apt install -y python3.7 python3-pip postgresql postgresql-contrib
-sudo apt install libpq-dev
+apt install -y python3.7 python3.7-dev python3-pip postgresql postgresql-contrib libpq-dev postgresql-common postgresql-client #postgresql-devel
 pip3 install --upgrade pip
-pip3 install -r /vagrant/requirements.txt
+
+# Installing pipenv, create new env and sync lib with pipfile
+pip3 install pipenv
+cd /vagrant/givitsite/
+pipenv run *
```

Comment:
not sure what are you trying to do here, run all the files in this directory before even running `sync` ?

Reply:
you right, i thought that this initial the ens but its not, anyway its work well without it (check), and erase this line and commit it bellow..